### DISCUSSION TITLE
Pomegrenade Extract does not give slow metabolism

### DISCUSSION THREAD

**Discussion by ReesesCups** [FakeFishGames/Barotrauma#13900] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

It was stated in patch notes that pomegrenade extract would give slow metabolism when injected, but I have repeatedly used it since the update was released and it simply does not have that effect. 

### Reproduction steps

1. Acquire a pomegrenade extract through your method of choice 
2. Inject the pomegrenade extract
3. Observe that there is no slow metabolism effect
4. cry

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multi player

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
It seems this line got accidentally added to the changelog: we considered adding this buff to pomegrenades, but ended up not adding it. Removed that line from the changelog now.
